### DISCUSSION TITLE
How can i change default background theme?

### DISCUSSION THREAD

**Discussion by kiordev** [TomSchimansky/CustomTkinter#1071] [Q&A]
Hello guys! My name is Alexandr. I am a student from Ukraine and speak english too bad so sorry for my gramatical mistakes.
Question:
How can i change main background color (system, dark, light - as default) to one i need using RBG code?
I found a json file where i can edit themes for button and other widgets, but i can not find this one.
Thank you for help early!

**Comment by TomSchimansky** [comment]
You can set the fg_color attribute for the CTk object with a hex color value:
```python
app = customtkinter.CTk(fg_color=("#hex-value-light", "#hex-value-dark"))
```
